applications: serial_lte_modem: add CONFIG_SLM_PPP_FALLBACK_MTU (#16802)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* Added:

  * DTLS support for the ``#XUDPSVR`` and ``#XSSOCKET`` (UDP server sockets) AT commands when the :file:`overlay-native_tls.conf` configuration file is used.

>   * Kconfig option :kconfig:option:`CONFIG_SLM_PPP_FALLBACK_MTU` to be able to control the MTU used by PPP when the cellular link MTU is not returned by the modem (with ``AT+CGCONTRDP=0``).

Reviewer: ```suggestion
  * The :kconfig:option:`CONFIG_SLM_PPP_FALLBACK_MTU` Kconfig option that is used to control the MTU used by PPP when the cellular link MTU is not returned by the modem in response to ``AT+CGCONTRDP=0`` AT command.
```
Check that the meaning is still correct. It should be, but I could've missed something.

Author: It is.
